transformers/events/pot_drip/converter.go

```diff
@@ -0,0 +1,43 @@
+package pot_drip
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+)
+
+type Converter struct{}
+
+func (c Converter) ToModels(_ string, logs []core.HeaderSyncLog, db *postgres.DB) ([]event.InsertionModel, error) {
+	var models []event.InsertionModel
+	for _, log := range logs {
+		err := shared.VerifyLog(log.Log, shared.TwoTopicsRequired, shared.LogDataNotRequired)
+		if err != nil {
+			return nil, err
+		}
+
+		msgSender := common.BytesToAddress(log.Log.Topics[1].Bytes()).Hex()
+		addressId, addressErr := shared.GetOrCreateAddress(msgSender, db)
+		if addressErr != nil {
+			_ = shared.ErrCouldNotCreateFK(addressErr)
```

Comment:
We probably want to return `addressErr` here

Reply:
Nice catch!